### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

**Comment by SimonCropp** [comment]
@Particular/nservicebus-maintainers merge please

**Comment by andreasohlund** [comment]
LGTM

File: src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs [hunk 1]
```diff
@@ -358,11 +358,16 @@ static string GenerateTestTimedOutMessage(TimeSpan maxTime)
...
-            await Task.WhenAll(runners.Select(r => r.InitializeTask).ToArray()).ConfigureAwait(false);
-            var failedRunner = runners.FirstOrDefault(r => r.InitializeTask.Result.Failed);
-            if (failedRunner != null)
+            var tasks = runners.Select(r => r.InitializeTask).ToArray();
+            await Task.WhenAll(tasks).ConfigureAwait(false);
+
+            foreach (var runner in runners)
             {
-                throw new ScenarioException($"Endpoint {failedRunner.Instance.Name()} failed to initialize", failedRunner.InitializeTask.Result.Exception);
+                var result = await runner.InitializeTask;
```
Comment (timbussmann) [inline src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs]: unnecessary await, but since the task is completed anyway it shouldn't really matter.

**Comment by timbussmann** [comment]
:+1:

File: src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs (hunk 1 above)
Comment (SimonCropp) [inline src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs]: yeah i thought about this one. my brain triggers when ever i see `.Result` so i went with the async. so if u want to go back to the result that is fine. but perhaps add a comment?

Comment (timbussmann) [inline src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs]: > my brain triggers when ever i see .Result

so does mine :+1: ;)

**Pull request merged**